BackofficeBundle/EventSubscriber/UpdateNodeRedirectionSubscriber.php

```diff
@@ -65,12 +65,25 @@ public function updateRedirection(NodeEvent $event)
     }
 
     /**
+     * @param NodeEvent $event
+     */
+    public function updateRedirectionRoutes(NodeEvent $event)
+    {
+        $node = $event->getNode();
+        $this->redirectionManager->updateRedirection(
+            $node->getNodeId(),
+            $node->getLanguage()
+        );
+    }
+
+    /**
      * @return array The event names to listen to
      */
     public static function getSubscribedEvents()
     {
         return array(
             NodeEvents::NODE_CHANGE_STATUS => 'updateRedirection',
+            NodeEvents::NODE_RESTORE => 'updateRedirectionRoutes',
```

Comment:
@SOLenG the `restore` term is linked to the name of the event launched. The `update` term is linked to the performed action.

IMHO, I think that this association is ok